.evergreen/config.yml

```diff
@@ -150,52 +126,48 @@ functions:
     - command: shell.exec
       params:
         continue_on_err: true
-        script: "set -o xtrace && rm -rf ${PROJECT_DIRECTORY}"
+        script: "set -o xtrace && rm -rf src"
     - command: s3.get
       params:
         aws_key: ${aws_key}
         aws_secret: ${aws_secret}
-        remote_file: ${UPLOAD_BUCKET}/${build_variant}/${revision}/${BUILD_NAME}/${build_id}.tar.gz
+        remote_file: ${project}/${build_variant}/${revision}/${task_name}/${build_id}.tar.gz
         bucket: mciuploads
         local_file: build.tar.gz
-    - command: shell.exec
-      params:
-        continue_on_err: true
-        # EVG-1105: Use s3.get extract_to: ./
-        script: "set -o xtrace && cd .. && rm -rf ${PROJECT_DIRECTORY} && mkdir ${PROJECT_DIRECTORY}/ && tar xf build.tar.gz -C ${PROJECT_DIRECTORY}/"
+        extract_to: src
 
   "exec compile script" :
     - command: shell.exec
       type: test
       params:
-        working_dir: "src"
         script: |
           ${PREPARE_SHELL}
-          [ -f ${PROJECT_DIRECTORY}/${file} ] && BUILDTOOL="${buildtool}" sh ${PROJECT_DIRECTORY}/${file} || echo "${PROJECT_DIRECTORY}/${file} not available, skipping"
+          [ -f ${file} ] && BUILDTOOL="${buildtool}" sh ${file} || echo "${file} not available, skipping"
```

Comment:
Are you sure this works?

The git checkout is into `src` and we want to execute the file from that working directory so that relative paths in the project directory work successfully.